### ISSUE TITLE
Videos no longer play (no 403 errors)

### ISSUE THREAD

**Issue by bigshiny90** [yuliskov/SmartTube#3739]
tried some videos this morning and nothing plays (morning july 22).  everything was working as of last night.

Tried latest beta and latest stable.  More youtube backend changes? or just me somehow? 

Tried multiple uninstalls/reinstalls...

**Comment by Adrian-dev-sys** [comment]
Same here, no error shows up and the videos are not playing. Changing the quality to 1080p doesn't help. I'm on the latest version. Not sure if I can check any logs to see what's wrong.

**Comment by bigshiny90** [comment]
Fixed it actually... related to latest 22.56 beta release. 

After updating to latest 22.56 release this morning, nothing played.  I then backed up data before deleting stn_beta and reinstalling - both stable latest (22.44) and previous beta (22.55).   Then of course restoring from backup.  No videos would play.

The fix was to Clear Data, uninstall current version, fresh install of Stable WITHOUT restoring from backup.  I then just re-added my account via normal google 'activate' process (not restore from backup).   everything is fine now.

Not sure what this is related to - if it is a problem with newest 22.56 - but whatever it was seemed to carry over with the backup/restore process.

interesting...

**Comment by emrk30** [comment]
> Fixed it actually... related to latest 22.56 beta release.
> 
> After updating to latest 22.56 release this morning, nothing played. I then backed up data before deleting stn_beta and reinstalling - i tried this with both stable latest (22.44) or previous beta (22.55). Then of course restoring from backup. No videos would play.
> 
> The fix was to Clear Data, uninstall current version, fresh install of Stable WITHOUT restoring from backup. I then just re-added my account via normal google 'activate' process (not restore from backup). everything is fine now.
> 
> Not sure what this is related to - if it is a problem with newest 22.56 - but whatever it was seemed to carry over with the backup/restore process.
> 
> interesting...

Not related to the latest beta .56 for me. I Tried a video this morning and got the no play all black no error code first, THEN I updated to beta .56 thinking it would help(I don't remember which version I was on but it was the first one that made 4k work after the 1080 fix for error 403) but of course the .56 update didn't work for me. So now I'm stuck and don't know to which version I should switch to or what else I can do

**Comment by Adrian-dev-sys** [comment]
> > Fixed it actually... related to latest 22.56 beta release.
> > After updating to latest 22.56 release this morning, nothing played. I then backed up data before deleting stn_beta and reinstalling - i tried this with both stable latest (22.44) or previous beta (22.55). Then of course restoring from backup. No videos would play.
> > The fix was to Clear Data, uninstall current version, fresh install of Stable WITHOUT restoring from backup. I then just re-added my account via normal google 'activate' process (not restore from backup). everything is fine now.
> > Not sure what this is related to - if it is a problem with newest 22.56 - but whatever it was seemed to carry over with the backup/restore process.
> > interesting...
> 
> Not related to the latest beta .56 for me. I Tried a video this morning and got the no play all black no error code first, THEN I updated to beta .56 thinking it would help(I don't remember which version I was on but it was the first one that made 4k work after the 1080 fix for error 403) but of course the .56 update didn't work for me. So now I'm stuck and don't know to which version I should switch to or what else I can do

I'm also on 22.44 stable and had this issue happen. What fixed it for me was clearing the data of the app in my system settings then logging back into my google account in smarttube. Unfortunately my settings were lost due to this, but videos work again. Strange bug indeed, it's probably related to the app's data or login session

**Comment by bigshiny90** [comment]
Yes… it’s looking like clearing app data and logging back in, seems key…

**Comment by QLaughable** [comment]
Aftering clearing data, videos play fine; however, after restoring my settings backup they no longer work

**Comment by bigshiny90** [comment]
Mentioned earlier in thread. Restoring from backup will break things again..

**Comment by chunkyjackson** [comment]
This didn't work for me. I did a fresh install of 22.57 logged in nothing would play. Upgraded to 22.59 again nothing would play, cleared the Data/Cache logged in again still nothing but error 403. Downgraded back to 22.38 everything works.

**Comment by emrk30** [comment]
Thanks guys, I cleared app data and it instantly worked for me, just had to relogin and reset every settings manually the way I like because backup makes it not work again. and still on .56 didn't update to the just released beta update yet.

**Comment by wilberfan** [comment]
Updated to 22.59 this morning, and I'm getting 403 errors on everything.  Just tried a clear data and re-login, but that didn't help.

**Comment by galmok** [comment]
I also just updated to 22.59 and now I just get 403. I could use a revert option. ;⁠-⁠)

**Comment by enko2974** [comment]
Doesnt work again

**Comment by Primeey** [comment]
22.59 and 22.44 stable don’t work for me. 403 error.

**Comment by wilsoncliu** [comment]
Same, not working for me after clearing data either.

**Comment by Quirkychai** [comment]
Not working for me either, latest beta. Tried stable, but only 1080p works.

**Comment by Netsky60** [comment]
I got 2 versions on my Shield : 
- 22.59 Beta version
- 22.38 stable version

I used the Beta version this morning (was 22.44), no issue, and this evening 403 errors started to appear. I upgraded the beta version to 22.59, same issues.
Tried Stable version, all is working fine on Nvidia Shied 👍 

I always keep a working version and i upgrade the other one to last version.

**Comment by ACR-Jeff** [comment]
On my Nvidia Shield TV
I had version 22.55 beta installed video player controls were showing yet videos weren't playing at all, 
Updated to version 22.59 beta and same thing, uninstalled, Went down the versions list installing each of them, After clearing cache/data and uninstalling, 
Reinstalling 22.55 beta and restoring the previous backup, works, I how ever do have to let it sit for about a minute and show the 403 on one video, After that it appears to work except on videos that were original live streams that ended, Those don't seem to play very well. Now trying a stable version, After testing 22.55 beta for some time it is hit and miss on what videos will play and won't play.
Went back to uninstalling 22.55 Beta and reinstalling without using the previous backup, Same thing as above mentioned.
Decided to try a stable version went all the way back to 22.38 stable without restoring previous backups, Appears to works except for recent previous live streams.

**Comment by jackL999** [comment]
>.. . Downgraded back to 22.38 everything works.

Thanks, this is the ONLY fix that worked for me.

**Comment by Reik57** [comment]
Setting a video preset to anything higher than 1080p makes it have the 403 error and eventually stop playing videos entirely. At some point after hard restarting the app (by force stopping it and opening it back in) it eventually plays a video, but you'll notice that anything higher than 1080p will be missing on the video quality options on the videos.

It seems we're back to 1080p for now. Setting a video preset to 4K seems to be the cause of it to have the 403 error.

Clearing data or reinstalling the app works because it removes the video presets you may have set before. If you have your settings backed up and have it as one of your settings, it may be the cause of the 403 error when restoring to backups.

I'm no expert though, these are things that I've just noticed on my Google TV when it suddenly stopped working for me.

Currently is working now for me on version 22.44. However, 4K option is gone. Tried setting 4K in video preset, didn't work, just gets me the 403 error, leading me to clear data and go back to default settings, worked for me but only with the option to watch until 1080p. I hope they find a fix again soon.

**Comment by xairoo** [comment]
Yeah, stopped working. Tested 22.44, 22.39 and 22.57 Beta.

**Comment by wioo** [comment]
Yeah beta doesnt work. I installed latest stable and it works (1080)

**Comment by xairoo** [comment]
> Yeah beta doesnt work. I installed latest stable and it works (1080)

22.44 stable doesn't work for me.

**Comment by ACR-Jeff** [comment]
Stable Version, Cleared cache/data and uninstalled, Reinstalled to the latest Stable release 22.44 works yet capped at 1080p

**Comment by Patraputt** [comment]
Smarttube is fully broken.

**Comment by ACR-Jeff** [comment]
> > Yeah beta doesnt work. I installed latest stable and it works (1080)
> 
> 22.44 stable doesn't work for me.

Didn't work for me either until after changing the preset to 1080p in the player settings, After that everything seemed to play after a restart of ST

**Comment by QLaughable** [comment]
22.38 Has no vp9 either

**Comment by Patraputt** [comment]
Everyone, please update to 22.65. It'll work like a charm!

**Comment by Timmyfox** [comment]
> Everyone, please update to 22.65. It'll work like a charm!
It didn't. Still having the same 403 errors like before.

Most commonly I'm getting the error at the start of my viewing session (i.e. when I first turn on the TV and go to view a video). Most recently (after updating to 22.65) I got a black screen right away for like 5-10 seconds or so until the video finally began playing and then after that no further errors for that session. Previously I've also experienced that the first or second video I watch suddenly goes black after a minute or two, together with the 403 error popping up, It'll then stay black for 5-10 seconds until the video again continues playing—also here it's usually once once at the beginning of my viewing session and after it happened once it won't happen again for the remainder of the session.

**Comment by josharghhh** [comment]
so had 22.65 working without issues until this morning. cant even get 1080
